target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -27,6 +27,9 @@ case "$board" in
 "glinet,ar150")
 	ucidef_set_led_wlan "wlan" "WLAN" "gl-ar150:orange:wlan" "phy0tpt"
 	;;
+"netgear,wndr3800")
+	ucidef_set_led_default "wan" "WAN LED (green)" "netgear:green:wan" "0"
```

Comment:
This one doesn't make any sense to me. The LED is already switched of during boot via `default-state = "off";`.

In case the intention is to have the LED - ready to configure - in /e/c/system, I've to say no. Considering big targets like ramips/ar71xx, it would add a massive amount of bloat.

If we want to have such a feature, it should be done on first boot by a script.

Reply:
Yeah, I can remove that. 
The similar item has been in ar71xx, so I replicated it here, but if the goal is to remove all script-configured LEDs from /etc/config, sure, I can remove that.

Reply:
The goal os rather to limit script configured LEDs to a minimum.